text/0000-variadic.md

```diff
@@ -76,6 +76,9 @@ impl<'a> VaList<'a> {
 impl<'a> Clone for VaList<'a>;
 ```
 
+`VaList` may become a language item (`#[lang="VaList"]`) to attach the
+appropriate compiler handling.
```

Comment:
`#[lang]` does not require RFC interactions, it's wholly internal to an implementation and done on a case-by-case need basis.

Reply:
That's why I didn't specify this precisely; I just mentioned it as a possibility. However, I moved this to the reference-level explanation, since that talks more about possible implementation details.